app/controllers/admin/podcasts_controller.rb

```diff
@@ -48,7 +48,7 @@ def update
     def fetch
       limit = params[:limit].to_i.zero? ? nil : params[:limit].to_i
       force = params[:force].to_i == 1
-      Podcasts::GetEpisodesWorker.perform_async(podcast_id: @podcast.id, limit: limit, force: force)
+      Podcasts::GetEpisodesWorker.perform_async("podcast_id" => @podcast.id, "limit" => limit, "force" => force)
```

Comment:
I'm honestly not sure if we should be using hashes at all. Is this really more readable than

```ruby
Podcasts::GetEpisodesWorker.perform_async(@podcast.id, limit, force)
```

IMO the keyword args for many of our jobs weren't the best choice to begin with, so if we have to make changes I'm tempted to move to positional args rather than option hashes. If only for the reason that there will be someone well-meaning wasting time on changing these back to symbol hashes in the future.

Reply:
@citizen428 I prefer keyword arguments over positional when there are more than 1-2 of them (I get them mixed up too easily). But I agree that hashes aren't the right tool for this, especially when we need to make them string keys. Feels really clunky.

Do we have a good place to discuss this outside of this PR?

Reply:
in this case (podcast get episodes worker) I'm aware of cases where we don't pass all three args (just the podcast id, with defaults for the other values), so it makes sense (or we'd have to change a few callers to pass all the arguments, or use optional positional arguments, which can become a problem as well).

I will say that _most_ of the jobs take integer id arguments or simple atomic values, so this PR is far shorter than it would have been.

Reply:
Since we have a great list now of jobs taking keyword arguments to perform I think calling API transition can be done in a separate PR?

On the plus side, making symbol keywords break when called will lead developers to avoid using hashes if unnecessary.